internal/dag/dag.go

```diff
@@ -145,8 +146,12 @@ func (d *DAG) Recompute() {
 	d.mu.Lock()
 	defer d.mu.Unlock()
 	version := d.dag.version
-	d.dag = d.recompute()
+	var status []ingressrouteStatus
+	d.dag, status = d.recompute()
 	d.dag.version = version + 1
+	for _, s := range status {
+		fmt.Printf("%v\n", s)
```

Comment:
This is fine for a proof of concept but we can't land this into master with this fmt.Printf in here as it will generate large amounts of log spam.